### DISCUSSION TITLE
Do you know that French official Open data website use Mljar ?

### DISCUSSION THREAD

**Discussion by julian-passebecq** [mljar/mljar-supervised#716] [General]
Hi,
I'm a french developer, i noticed recently that french open data is using mljar for auto ml.
Is it a partnership ? Does the french agency responsible for the usage has to pay some kind of license ? I'm really curious, and thrilled that a tool i'm using is gaining popularity :) This website is recognized by the government and has official datasets about economy for ex.
![image](https://github.com/mljar/mljar-supervised/assets/64646018/df0c5b32-9828-47b0-abcb-81e51eaf72cd)
Translation : 
`Here we offer a catalog of datasets identified as usable by machine learning algorithms grouped by task. Each game is accompanied by the results of training and automatic testing of classic Machine Learning algorithms: metrics, confusion matrices, graphs, etc.

N.b.: This page is under construction and is open to contribution, more datasets are to come. The models were trained using mljar-supervised.`

**Comment by pplonski** [comment]
Hi @julian-passebecq,

Thank you for information. We are super happy that mljar-supervised is used by many users. We don't have any partnership with French gov. They are just using our open-source package.

We hope that mljar-supervised will be even more popular -  we are working on a new way for writing python programs by interactive code recipes.  Here is [docs for AutoML training](https://mljar.com/docs/python-train-automl/) - docs are interactive :) We hope to release first version of MLJAR Studio this month :)
